corehq/ex-submodules/couchforms/_design/filters/xforms.js

```diff
@@ -11,10 +11,12 @@ function(doc, req)
     switch (doc_type) {
         case "XFormInstance":
         case "XFormArchived":
+        case "XFormError":
         case "XFormDeprecated":
         case "XFormDuplicate":
         case "XFormInstance-Deleted":
         case "HQSubmission":
+        case "SubmissionErrorLog":
             return true;
         default:
             return false;
```

Comment:
I would guess that what @czue said is right, but I actually haven't ever tested that. It'd be pretty easy to throw this up on staging and see if it causes a reindex though! (Even the amount of data we have on staging is usually enough to cause a > 5-min preindex.)